lib/ok_computer/built_in_checks/app_version_check.rb

```diff
@@ -5,6 +5,21 @@ module OkComputer
   # * Otherwise, checks for Capistrano's REVISION file in the app root.
   # * Failing these, the check fails
   class AppVersionCheck < Check
+    attr_accessor :file
+    attr_accessor :env
+    attr_accessor :transform
+
+    # Public: Initialize a check for a backed-up Sidekiq queue
+    #
+    # file - The path of the version file to check
+    # env - The key in ENV to check for a revision SHA
+    # transform - The block to optionally transform the version string
+    def initialize(file = "REVISION", env = "SHA", &transform)
```

Comment:
I think I'd rather use [keyword arguments ](https://thoughtbot.com/blog/ruby-2-keyword-arguments) for this. That way the position of the argument or their existence doesn't matter. Like this:
```suggestion
    def initialize(file: "REVISION", env: "SHA", &transform)
```

Reply:
@andrewschaar Agreed on the keyword args. I would have actually done that from the get-go, but it didn't look like any existing checks used them. I'll switch.

Reply:
Switched to keyword args. Let me know if you'd like any other modifications. Thanks.